src/nvim/CMakeLists.txt

```diff
@@ -357,6 +357,14 @@ foreach(hfile ${NVIM_GENERATED_FOR_HEADERS})
   endif()
 endforeach()
 
+# Add xdiff sources to the build as-is bypassing the checks and linters
+file(GLOB XDIFF_SOURCES xdiff/*.c)
+file(GLOB XDIFF_HEADERS xdiff/*.h)
+if(NOT MSVC)
+  set_source_files_properties(
+    ${XDIFF_SOURCES} PROPERTIES COMPILE_FLAGS "${COMPILE_FLAGS} -Wno-conversion")
```

Comment:
Instead of doing this, would it make sense to add the files to `CONV_SOURCES` (line 132)?

Reply:
Probably, I should agree for the sake of code reuse.